examples/include/crypto-accelerator.p4

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-/// Crypto accelerator Extern definition
+// BEGIN:Crypto_accelerator_extern_object
```

Comment:
This will quote over 100 lines in the document - maybe this marker can be moved to line 82 (just before the `extern` keyword), and readers can come to this source file if they want more details?

(Don't feel strongly about this one.)